influxdb/1.5/init-influxdb.sh

```diff
@@ -11,8 +11,9 @@ fi
 
 INIT_USERS=$([ ! -z "$AUTH_ENABLED" ] && [ ! -z "$INFLUXDB_ADMIN_USER" ] && echo 1 || echo)
 
-if ( [ ! -z "$INIT_USERS" ] || [ ! -z "$INFLUXDB_DB" ] || [ "$(ls -A /docker-entrypoint-initdb.d 2> /dev/null)" ] ) && [ ! "$(ls -d /var/lib/influxdb/meta 2>/dev/null)" ]; then
+if [ -z $INFLUXDB_META_DIR ]; then META_DIR="/var/lib/influxdb/meta" else META_DIR=$INFLUXDB_META_DIR fi
```

Comment:
Can you break this up into multiple lines and add a comment? It's just hard for me to read it.

Reply:
sure